datafusion/core/tests/sqllogictests/src/engines/conversion.rs

```diff
@@ -87,5 +87,5 @@ pub fn decimal_to_str(value: Decimal) -> String {
 }
 
 pub fn big_decimal_to_str(value: BigDecimal) -> String {
-    value.round(12).normalized().to_string()
+    value.round(16).normalized().to_string()
```

Comment:
`Sqrt(2)` returns a double value of `1.4142135623730951`. 
the precision of `value.round(12)` is not sufficient, `1.414213562373` is returned. 
To improve accuracy, we could increase the rounding precision to `value.round(16)`

Reply:
will cause other testcase failed,so  i reverted change.